ui_framework/src/components/call_out/_call_out.scss

```diff
@@ -0,0 +1,57 @@
+.kuiCallOut {
+  padding: $kuiSizeM $kuiSizeL;
+  border-left: 4px solid $kuiColorPrimary;
+  background-color: rgba($kuiColorPrimary, 0.1); /* 1 */
```

Comment:
I think we should change our approach here so that components with modifiers _require_ a modifier to take on an appearance. In other words, we'd remove these default values and add an `info` definition to `$callOutTypes`, below (and do something similar with `KuiButton` and `KuiIcon`).

I kept running into cascade problems with KuiIcon because of its default `fill`, and this approach avoids that problem. It also makes it easier to scan the modifiers and know exactly what kind of use cases a component addresses.